plugins/modules/github_tag.py

```diff
@@ -0,0 +1 @@
+./source_control/github/github_tag.py
```

Comment:
Judging from the CI errors, I guess this is not a relative symlink, but a text file with the relative filename as content? It must be a symlink for everything to work.

Reply:
Thanks I got a little confused I think about the top level files !